lsn10:
  ros__parameters:
    scan_topic: scan
    lidar_frame: laser_link
    serial_port: /dev/ttyUSB0
